README.md

```diff
@@ -1 +1,161 @@
-# to-do-list
\ No newline at end of file
+# to-do-list
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [WebPack](#)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [to-do-list] <a name="to-do-list"></a>
+
+**[to-do-list]** is used to  display a to do list 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML  ,CSS and JavaScript.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[To do List]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- This Porject is not deployed yet.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+Node.js, git , linters (Lighthouse, Webhint and Styleint) and a code editor.
+
+--
+Example command:
+
+```
+  npm install -g lighthouse
+  npm install --save-dev hint@7.x
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+  npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+```
+ --
+
+### Setup
+
+Clone this repository to your desired folder:
+
+--
+Example commands:
+
+```
+  cd my-folder
+  git clone https://github.com/MustafaSamiShirzad/to-do-list
+```
+---
+
+### Install
+
+Install this project with: git
+
+### WebPack Plugins
+-html-webpack-plugin
+-sass css-loader style-loader sass-loader
+-web dev server
+### Run tests
+
+To run tests, run the following command:
+
+--
+Example command:
+
+```
+  npx hint .
+  npx stylelint "**/*.{css,scss}"
+  npx eslint .
+```
+---
```

Comment:
- [ ] @MustafaSamiShirzad Great work on your documentation, however, you forgot to write instructions on how to run the project locally eg initialize using `npm i` the run the app using `npm run dev`, also I would recommend the change of  content in running test since the commands are for linter checks and not running tests in the app.

Reply:
Ok, I will add these commands